.github/workflows/staging-release.yml

```
@@ -15,17 +15,32 @@ jobs:
       - name: Checkout Repo
         uses: actions/checkout@v2
 
-      - name: Setup Node.js 12.x
+      - name: Setup Node.js 16.x
         uses: actions/setup-node@v2
         with:
-          node-version: 12.x
+          node-version: 16.x
+
+      - name: Creating .npmrc
+        run: |
+          cat << EOF > "$HOME/.npmrc"
+            //registry.npmjs.org/:_authToken=$NPM_TOKEN
+          EOF
+        env:
+          NPM_TOKEN: ${{ secrets.NPM_TOKEN }}
+          
+      - name: Configure npm node prepend 
+        run: npm config set scripts-prepend-node-path auto
 
       - name: Install Dependencies
         run: yarn
 
-      - name: Create Release Pull Request
-        uses: changesets/action@v1
-        with: 
-          version: yarn version:staging
+      - name: Install Dependencies build
+        run: yarn build
+
+      - name: version staging packages
```

comment: This is very nit, but just want to stay consistent 😅 
```suggestion
      - name: Version staging packages
```